From 1a70f76671794c17cc2f5038310cd2ef81c4464f Mon Sep 17 00:00:00 2001
From: Hieu Huynh <hieuh@os.amperecomputing.com>
Date: Tue, 30 Nov 2021 04:55:40 +0000
Subject: [PATCH] Add d-bus OEM for boot progress

This commit adds the BootProgressOEM for the OEM-specific LastState
as defined in the DEN0069C_SBMR_1.1 document.

Signed-off-by: Hieu Huynh <hieuh@os.amperecomputing.com>
---
diff --git a/yaml/xyz/openbmc_project/State/Boot/Progress.interface.yaml b/yaml/xyz/openbmc_project/State/Boot/Progress.interface.yaml
index b064f12..d2759a4 100644
--- a/yaml/xyz/openbmc_project/State/Boot/Progress.interface.yaml
+++ b/yaml/xyz/openbmc_project/State/Boot/Progress.interface.yaml
@@ -18,6 +18,12 @@ properties:
           BootProgress value to know how long the boot has been on the current
           boot step.
 
+    - name: BootProgressOem
+      type: string
+      description: >
+          This property shall represent the OEM-specific LastState of
+          the BootProgress.
+
 enumerations:
     - name: ProgressStages
       description: >
-- 
2.17.1

